Question 1.js

```diff
@@ -0,0 +1,13 @@
+

```

Comment:
How did you use this file to display output on the webpage? I don't see a HTML file.